time stamp
string agent
uint16[] seqs
bool clear
bool reset
bool hardReset
bool ma_reset
